mmv1/products/bigqueryconnection/Connection.yaml

```diff
@@ -110,6 +110,13 @@ examples:
     vars:
       connection_id: "my-connection"
       database: "projects/project/instances/instance/databases/database"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "bigquery_connection_spark"
+    pull_external: true
```

Comment:
`pull_external` shouldn't be needed. It's only needed if an external provider to the example is needed to execute the test. Like `random`

Reply:
Sorry. copy pasta mistake. will remove